app/components/post_list/post_list.js

```diff
@@ -313,6 +313,7 @@ export default class PostList extends PureComponent {
                 managedConfig={managedConfig}
                 onLayoutCalled={this.measureItem}
                 shouldCallOnLayout={this.props.measureCellLayout && !this.newMessageScrolledTo}
+                showAddReaction={true}
```

Comment:
Maybe is a good idea to add a default prop as true so you don’t need to do this everywhere, and just in case if you missed one or two

Reply:
Thanks! Updated with default props to true.